REDHAT.rst

```diff
@@ -0,0 +1,45 @@
+.. |br| raw:: html
+
+   <br />
+
+.. Links
+.. _download page: https://foglamp.readthedocs.io/en/master/92_downloads.html
```

Comment:
This has been changed to http://dianomic.com/download-packages/ Refer docs/92_downloads.rst

Reply:
done